test/source/tests/tests/compose.ts

```diff
@@ -16,6 +16,22 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
 
   if (testVariant !== 'CONSUMER-LIVE-GMAIL') {
 
+    ava.default('compose[global:compatibility] - toggle minimized state by clicking compose window header', testWithNewBrowser(async (t, browser) => {
```

Comment:
Here `testWithNewBrowser` is super slow because it has to spin up a new browser, log into an account, set up the extension for each test.

When you are not changing any account settings, better use `testWithSemaphoredGlobalBrowser`. This will run the test on a recycled browser that already has an account set up, and then release the browser to be used for other similar tests - much faster.

You can choose which account you need for the test, eg `compatibility`.